<launch>

	<!-- top and bot wb sensors -->
	<include file="$(find dwyer_sensor_driver)/launch/wbBox.launch" /> 

	### Plume source from Box ###
	#-------------------------------#
	<param name="biasC" type="int" value="100"/>
	<param name="loopR" type="int" value="100"/>
	<param name="tc" type="double" value=".1"/>
	<node pkg="dwyer_sensor_driver" type="dwyer_sensor" name="dwyer_sensor" output="screen"/>
	#-------------------------------#
	<param name="Kl" type="double" value="0.5"/>
	# in inches #
	<param name="radi" type="double" value="3"/>	
	<node pkg="dwyer_sensor_driver" type="release_vel" name="release_vel" output="screen"/>
	#-------------------------------#
	
</launch>